### PR TITLE
[MRG][FIX] Fixes to psd_topo and added megsim data download to circle.

### PR DESCRIPTION
**Pull request by jaeilepp**
I don't know when ``plot_psd_topo`` stopped working, but here is a fix to it. Also,  seems that megsim downloading wasn't included in circle, so it's included here.

### CODE REVIEW

**Comment by jaeilepp** [comment]
Test updated. I wonder if we should at some point update the testing data to have ias channels.
Ready for review.

**Comment by agramfort** [comment]
thx @jaeilepp

File: mne/viz/tests/test_raw.py [hunk 1]
```diff
@@ -25,6 +25,8 @@
 
 def _get_raw():
     raw = io.Raw(raw_fname, preload=True)
+    with warnings.catch_warnings(record=True):
```
Comment (Eric89GXL) [inline mne/viz/tests/test_raw.py]: can you in `master` add a comment on this line or above it saying what the warning was for? it's not immediately obvious to me from the code

Comment (jaeilepp) [inline mne/viz/tests/test_raw.py]: Done.

**Pull request merged**